src/pages/home/ReportScreen.tsx

```diff
@@ -95,7 +96,8 @@ function getParentReportAction(parentReportActions: OnyxEntry<OnyxTypes.ReportAc
     return parentReportActions[parentReportActionID ?? '0'];
 }
 
-function ReportScreen({route, currentReportID = '', navigation}: ReportScreenProps) {
+function ReportScreen({currentReportID = '', navigation}: ReportScreenProps) {
+    const route = useRoute<RouteProp<AuthScreensParamList, typeof SCREENS.REPORT>>();
```

Comment:
Why is this change necessary?

Reply:
But this is already working on `main`, what broke it?

https://github.com/user-attachments/assets/06481aa5-ab8d-43ba-b792-da97bc5b42a8

Reply:
@tsa321 Can you please clarify the solution independently for those two bugs:
- Not showing "new message" button
- Not navigating on clicking same report action link


(Also please merge `main`)